foxglove/cmd/utils.go

```diff
@@ -150,11 +150,37 @@ func AddFormatFlag(cmd *cobra.Command, format *string) {
 		format,
 		"format",
 		"",
-		"table",
+		"",
+		"render output in specified format (table, json, csv)",
+	)
+}
+
+func AddJsonFlag(cmd *cobra.Command, isJsonFormat *bool) {
+	cmd.PersistentFlags().BoolVar(
```

Comment:
Not a big deal, but if this is a boolean, what happens with ` --json=false --format=json`?

Reply:
this will output json!
i didn't even think of doing `--json=false` 😂
should i change this behaviour?

Reply:
I don't suppose cobra has an option for a boolean var that doesn't take an arg/can't be set to false...

I would not worry about this. (When I read the CLI help, I do not even assume this is possible — I just wondered because of the source.)